specs/opcode/15ISZERO.md

```diff
@@ -0,0 +1,36 @@
+# ISZERO opcodes
+
+## Procedure
+
+### EVM behavior
+
+Pop an EVM word `value` from the stack. If it is zero, push `1` back to the stask. Otherwise push `0` to stack.
+
+### Circuit behavior
+
+The IsZeroGadget takes argument of `value: [u8;32]` and `result: [u8;32]`.
+
+If `value` is zero, it validates `result === 1`. Otherwise it validates `result === 0`.
+
+We annotate stack as \[value, ...\] and \[result, ...\].
+
+## Constraints
+
+1. opId = OpcodeId(0x15)
```

Comment:
remove this line, there's no opcodeId check contraints  in `ISZERO` circuit.

1. your implementation doesn't have it
2. we don't necessarily need it, see `02MUL.md`
3. `01ADD_03SUB.md` needs it because it deals with 2 different op (`ADD` & `SUB`) at the same time